Keas.Mvc/ClientApp/pages/assets/AssetNav.tsx

```diff
@@ -14,7 +14,7 @@ export default class AssetNav extends React.Component<{}, {}> {
               className='tab-link tab-people'
               activeClassName='active'
             >
-              People
+              Peoples
```

Comment:
@sprucely Do we really want to change this? I guess we should ask Cal but we've always used People as the plural in this app, not peoples

Reply:
Not something I would deliberately do. I wonder how it happened,

Reply:
Oh, now I remember. I was testing the hot reloading and forgot to undo it.